scripts/install_all.sh

```diff
@@ -165,6 +165,16 @@ using an additional ~185MB of disk space."
     fi
 fi
 
+if ! command -v isort > /dev/null; then
+    echo "Installing isort..."
+    if [ $APT ] || confirm "Would you like to automatically install isort?"; then
+        pip3 install isort seed-isort-config
+    else
+        echo "You need to manually install isort (code formatter) before continuing..." >&2
+        exit 1
+    fi
+fi
```

Comment:
I thought `pre-commit` was going to handle installation of its packages for us?

Reply:
it does, forgot to remove this.

Reply:
done